lib/caasp.pm

```diff
@@ -20,11 +20,47 @@ use version_utils 'is_caasp';
 use utils qw(power_action assert_shutdown_and_restore_system);
 
 our @EXPORT = qw(
-  trup_call trup_install rpmver process_reboot check_reboot_changes microos_login
+  trup_call trup_install rpmver process_reboot check_reboot_changes microos_login send_alt
   handle_simple_pw export_cluster_logs script_retry script_run0 script_assert0
   get_delayed update_scheduled
   pause_until unpause);
 
+# Shorcuts for gui / textmode (optional) oci installer
+my %keys = (
+    '4.0' => {
+        keyboard     => ['y', 'e'],
+        password     => ['a', 'a'],
+        registration => ['g'],
+        role         => ['s'],
+        partitioning => ['p'],
+        booting      => ['b'],
+        network      => ['n'],
+        kdump        => ['k'],
+        install      => ['i'],
+        ntpserver    => ['r'],
+    },
+    '3.0' => {
+        keyboard     => ['e', 'y'],
+        password     => ['w', 'a'],
+        registration => ['g'],
+        role         => ['s'],
+        partitioning => ['p'],
+        booting      => ['b'],
+        network      => ['n'],
+        kdump        => ['k'],
+        install      => ['i'],
+        ntpserver    => ['t'],
+    });
+
+# Send alt shortcut by name
+sub send_alt {
+    my $key = shift;
+    my $ver = get_var('VERSION');
+    my $txt = check_var('VIDEOMODE', 'text');
+
+    send_key "alt-$keys{$ver}{$key}[$txt]";
```

Comment:
As per documentation check_var returns `undef` (http://open.qa/api/testapi/), but in the code I acknowledge that it's 0 or 1.

Reply:
I should read our doc when I have some time :) - but as you said, it's wrong in this case..